app/helpers/ui_helper.rb

```diff
@@ -23,7 +23,8 @@ def button(label: nil, type: 'primary', size: 'md', icon_name: nil, icon_positio
         icon_position:,
         tooltip_text:,
         tooltip_position:,
-        disabled:
+        disabled:,
+        html: html || {}
```

Comment:
Just

`html: {}` is enough

rename html to html_options, that is the standard naming